packages/backend/src/plugins/catalog.ts

```diff
@@ -14,6 +14,7 @@ import {
   AWSDynamoDbTableDataProvider,
   AWSIAMRoleProvider,
   AWSIAMRoleProcessor,
+  AWSEKSClusterProvider,
```

Comment:
This adds a new entity provider. It'll go to AWS and list the aws clusters and create entities in the backstage database for each.